src/Symfony/Component/Notifier/Bridge/MessageBird/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/message-bird-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony MessageBird Notifier Bridge",
+    "keywords": ["sms", "message-bird", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Vasilij Duško",
+            "email": "vasilij@prado.lt"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
```

Comment:
I would use `^4.4|^5.2` as previous versions are not maintained anymore.

Reply:
@fabpot thanks. Fixed.